content/docs/conditional-rendering.md

```diff
@@ -8,11 +8,11 @@ redirect_from:
   - "tips/false-in-jsx.html"
 ---
 
-In React, you can create distinct components that encapsulate behavior you need. Then, you can render only some of them, depending on the state of your application.
+React में, आप अलग-अलग कौम्पोनॅन्ट बना सकते हैं जो आपके लिए आवश्यक व्यवहार को संक्षिप्त करते हैं। फिर, आप अपने ऍप की स्टेट के आधार पर उनमें से कुछ को ही रिप्रेजेंट कर सकते हैं।
```

Comment:
We were not translating encapsulate if I remember correctly, @arshadkazmi42  ?

Reply:
@saranshkataria Yes, right. We were just writing the hindi version of it. I think we should add this in #2 also.

Reply:
done

Reply:
@kambleaa007 we are writing it as एनकैप्सुलेट

Reply:
Done